functions/go/create-setters/README.md

```diff
@@ -0,0 +1,136 @@
+# create-setters
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Create setter values on resource fields. Setters serve as parameters for template-free
+setting of field values.
+
+<!--mdtogo-->
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+Setters are a safer alternative to other substitution techniques which do not
+have the context of the structured data. Setters may be invoked to modify the
+package resources using `create-setters` function to set desired values.
+
+We use ConfigMap to configure the `create-setters` function. The desired setter
+values are provided as key-value pairs using `data` field where key is the name of the
+setter and value is the new desired value for the setter.
```

Comment:
Try to understand the difference between apply-setters and this function and modify this README. Most of it looks good but some parts looks like copy-paste from apply-setters.